docs/SetupDatabaseForAWS.md

```diff
@@ -0,0 +1,31 @@
+# Set up a database for Scalar DB/Scalar DL deployment on AWS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment on AWS.
+
+## DynamoDB
+
+Amazon DynamoDB is available for all AWS users. You do not need to set up anything manually.
```

Comment:
What is `Amazon DynamoDB is available for …` sentence for?
Are we explaining who a product is available for each database?

Reply:
> What is Amazon DynamoDB is available for … sentence for?

It is to explain that DyanmoDB is available for all users and we don't have to create it.

>Are we explaining who a product is available for each database?

No, I will update the sentence to make it more clear.
Sorry for the confusion.

Reply:
I have updated the sentence.